rfcs/0013-synthea-future-claims-and-automation.md

```diff
@@ -0,0 +1,139 @@
+
+  - Weekly - Synthetic data will be generated with a set batch size using the master branch of the Synthea codebase in a hosted cloud instance, along with the Synthea end-state properties file from the most recent generated batch. The end-state properties file is critical for making sure the next batch of Synthea data will not overlap with released Synthea data.
+
+  - Quarterly - The main difference in the process of generating Synthea data each quarter will be using the most recent version of the master branch of the Synthea codebase. This should introduce contrast in beneficiary and claim data properites between quarterly and weekly generated data, as Mitre regularly makes changes to the output of Synthea data generated over time. 
+
+  - Future Claim Data - For both quarterly and weekly, a batch of similar size for claim data with a future date will also be generated in a similar fashion. 
+
+  - Special data parameters of interest: One parameter, which has not changed in previous releases of Synthea is Part D Event ID (PDE ID). Despite monotonically increasing other parameters i.e. bene ID, claim group ID, the value of PDE ID will not need to change on a regular basis, unless there is a specific customer use case presented
+
+Automated Recurring Loading: 
+  - Once the synthetic data is generated, within the same hosted cloud instance, the latest version of the BFD application from github, and dockerized database, will have been installed prior, and will be used to run the BFD pipeline integration test (IT), to load the new batch of synthetic data, ensuring that the newly generated data can be inserted properly.
+
+  - Additionally, the RIF files from the previous weekly/quarterly batch of Synthea data will be pulled from AWS S3, and loaded into the database with the new batch of synthea data.  
+
+  - Once the automated load testing on the instance is complete, a script will run that modifies the new batch's generated manifest file, and uploads the RIF files, end-state properties, and manifest files to S3 via command line file transfer. This will trigger the BFD ingestion pipeline in PROD SBX or TEST so that the data is stored in the database.
+
+  - The separate RIF files associated with future claim data for the given release will be put uploaded in a specially designated folder in S3 for staging until loading in the future. 
+
+  - Additional testing - As of 04/21/2022, there is a plan in place to remve tens of thousands of old synthetic beneficiary data prior to Synthea data being stored in the database, which is largely responsible for the database collisions that have taken place in the past, when loading new Synthea data. Once this data is permenantly removed, it will likely take many years for there to be enough data to cause overlap. So after making sure the data generated does not collide with the previous batch of Synthea data, it should be safe to load.  
```

Comment:
✏️  typo on the first line "remove"

Also minor clarification:
In test and prod-sbx, there are 50k synthetic benes and 6 million de-duped non-synthetic benes, along with ~10-15 3 digit benes that were used for some other testing in the past. The cleanup will be removing the 6million non-negative non-synthetic benes, along with the 3 digit benes, but will be leaving the existing 50k synthea benes. Any issues with collision will probably not be helped by this cleanup, since its primarily removing non-negative benes. (It also means we need to start our automated bene creation at the current bene id number to avoid collisions)

I updated the https://github.com/CMSgov/beneficiary-fhir-data/wiki/Synthetic-Data-Guide to show the ranges that will be saved (plus one for the golden bene)